docs/developer/continuous-deployment/continuous_deployment.md

```diff
@@ -0,0 +1,200 @@
+# Continuous Deployment
```

Comment:
why git delete and add instead of move? We could see the diff then

Reply:
Not related to this PR. Was a rebase issue.